### ISSUE TITLE
DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.

### ISSUE THREAD

**Issue by siketyan** [VOICEVOX/voicevox_engine#646]
```
python run.py --enable_mock
```

says:

```
run.py:1208: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  latest_core_version = str(max([LooseVersion(ver) for ver in synthesis_engines]))
```

**Comment by siketyan** [comment]
Maybe closed by #609

**Comment by aoirint** [comment]
closed by #609
